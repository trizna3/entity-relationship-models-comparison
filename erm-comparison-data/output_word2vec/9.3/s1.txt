payments-contract
account;students-students
buildings-buildings
contracts-contracts
rooms-rooms
EXTRACT_ATTR_TO_OWN_ENTITY_SET:S-contract-students
GENERALIZATION_TO_11_ASSOCIATION:S-contracts-rooms
CONTRACT_11_ASSOCIATION:E-account-students
