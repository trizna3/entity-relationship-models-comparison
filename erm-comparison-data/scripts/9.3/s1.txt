entity set addresses {
	"name"
};

entity set buildings {
	"address"
	"floor"
};

entity set rooms {
	"capacity"
	"monthly fee"
};

entity set contracts {
	"start"
	"end"
};

entity set students {
	"contract"
	"credit"
};

association between 
{
	addresses ONE 
	buildings ONE
}{};

association between 
{
	buildings ONE
	rooms MANY
}{};

association between 
{
	contracts ONE 
	students MANY
}{};

generalization from contracts to rooms;