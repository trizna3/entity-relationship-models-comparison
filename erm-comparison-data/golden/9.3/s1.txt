buildings-buildings
rooms-rooms
contracts-contracts
payments-EMPTY
students;account-students
CONTRACT_11_ASSOCIATION:E-students-account